docs/README.md

```diff
@@ -0,0 +1,21 @@
+### How to generate doc
+
+```bash
```

Comment:
Scriptize

Reply:
Temporal solution for now.
https://github.com/ki4070ma/python-client-sphinx/blob/master/github_pages_deploy.sh